### PR TITLE
Revised Jira integration page to be more task-based

### PR DESCRIPTION
**Pull request by robpearson**

### CODE REVIEW

File: docs/api-and-integration/metadata/jira.md [hunk 1]
```diff
@@ -3,69 +3,69 @@ title: Jira Issue Tracking Integration
...
-The Octopus Deploy plugin for Jira is only compatible with Jira Software Cloud as Jira Server (on-prem) does not support the APIs required to enable this functionality. There is limited support available as noted in our [Jira Server section](#jira-server-on-prem-support).
+The Octopus Deploy plugin for Jira is only compatible with Jira Software Cloud as Jira Server (on-premises) does not support the APIs required to enable this functionality. There is limited support available as noted in our [Jira Server section](#jira-server-on-prem-support).
 :::
 
 ## Connecting Jira and Octopus Deploy
 
-1. Install the Octopus Deploy plugin in Jira
+1. Install the Octopus Deploy plugin in Jira.
 
     Add the `Octopus Deploy for Jira` app from the [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and then click the 'Get Started' button to configure it.
```
Comment (wordlee) [inline docs/api-and-integration/metadata/jira.md]: Do we need to provide orientation here? i.e., From within your Jira administration dashboard...

File: docs/api-and-integration/metadata/jira.md [hunk 2]
```diff
@@ -3,69 +3,69 @@ title: Jira Issue Tracking Integration
...
 
-    Navigate to **{{Infrastructure,Environments}}** to map your Octopus environments to Jira environment types. This is required so Jira is able to understand Octopus environments and track issue progress. 
-    
-    Note: Jira environment types are a fixed list that cannot be edited. 
+    Navigate to **{{Infrastructure,Environments}}** to map your Octopus environments to Jira environment types. This is required so Jira can understand Octopus environments and track issue progress.
 
-The end result of this integration is to provide Jira with updates on the progress of Jira issues (work items) through the pipeline.
+    Note: Jira environment types are a fixed list that cannot be edited.
+
+When configured this integration will provide Jira with updates about the progress of Jira issues (work items) through the pipeline.
```
Comment (wordlee) [inline docs/api-and-integration/metadata/jira.md]: For the sake of this audience, I don't know that we need to include the phrase (work items).

File: docs/api-and-integration/metadata/jira.md [hunk 3]
```diff
@@ -1,75 +1,71 @@
...
+    Note: Jira environment types are a fixed list that cannot be edited.
 
-The environment name also gets passed through to Jira, as shown in the earlier example. Also note that multi-tenancy is fully supported and the deployment will show in Jira per tenant per environment name.
+When configured this integration will provide Jira with updates about the progress of Jira issues (work items) through the pipeline.
 
-The following diagram illustrates an un-tenanted deployment to Dev and then a tenanted deployment to Dev of the same version (the text is truncated in the screenshot, the alt text for the link reads `Deploy work items test release 0.0.5 to Dev for Tenant A`)
+## Octopus Deployment Task Log
 
-![Jira Multi-Tenant progress](jira-multi-tenant.png)
+When the Jira issue tracker is enabled and configured, you will see blocks similar to the following appear in the log during your deployments. These show the state updates Octopus is sending through to Jira, and if you expand them the details include information about the Jira issues for traceability.
```
Comment (wordlee) [inline docs/api-and-integration/metadata/jira.md]:
```suggestion
When the Jira issue tracker is enabled and configured, the task log will show the state updates Octopus is sending through to Jira. If you expand them the details include information about the Jira issues for traceability.
```

File: docs/api-and-integration/metadata/jira.md (hunk 1 above)
Comment (robpearson) [reply docs/api-and-integration/metadata/jira.md]: Yes, that's probably a good idea. It's not hard but we should probably add more detail.

File: docs/api-and-integration/metadata/jira.md (hunk 2 above)
Comment (robpearson) [reply docs/api-and-integration/metadata/jira.md]: Agreed.

**Comment by robpearson** [comment]
This is out of date. Closing.

**Pull request closed without merging**